tests/tests_geomstats/test_riemannian_mean_shift.py

```diff
@@ -0,0 +1,40 @@
+""" Unit tests for Riemannian Mean Shift method """
+
+import numpy as np
+
+import geomstats.tests
+from geomstats.geometry.hypersphere import Hypersphere, HypersphereMetric
+from geomstats.learning.frechet_mean import FrechetMean
+from geomstats.learning.riemannian_mean_shift import RiemannianMeanShift as RMS
+
+
+@geomstats.tests.np_and_autograd_only
```

Comment:
Do you encounter an error when removing this decorator, and if so which one? Thanks!

Reply:
No, I just removed the decorator and the test case works fine.